bundles/org.eclipse.ui.themes/css/dark/e4-dark_partstyle.css

```diff
@@ -108,7 +108,7 @@
 }
 .MPart FormHeading > CLabel {
     background-color: #505f70;
-    color: #E98787;
```

Comment:
setting this always to "red" indeed was wrong (as we could also display warnings and not only errors) there.